app/models/container.rb

```diff
@@ -27,16 +27,17 @@ class Container < ApplicationRecord
 
   acts_as_miq_taggable
 
-  def event_where_clause(assoc = :ems_events)
-    case assoc.to_sym
-    when :ems_events, :event_streams
-      # TODO: improve relationship using the id
-      ["container_namespace = ? AND #{events_table_name(assoc)}.ems_id = ? AND container_name = ?",
-       container_project.try(:name), ext_management_system.try(:id), name]
-    when :policy_events
-      # TODO: implement policy events and its relationship
-      ["#{events_table_name(assoc)}.ems_id = ?", ext_management_system.try(:id)]
-    end
+  def event_where_clause_ems_events
+    # TODO: improve relationship using the id
```

Comment:
Strange...is there no container_id on the table?

Reply:
or maybe it is target_type target_id?

Reply:
Yeah, it's weird. I don't know why it was done this way.  I'll try to investigate it.